packages/apps/plugins/plugin-sketch/src/hooks/adapter.ts

```diff
@@ -139,6 +212,9 @@ export class AutomergeStoreAdapter implements StoreAdapter {
 
       // Update/remove the records in the store.
       store.mergeRemoteChanges(() => {
+        if (updated.size) {
+          store.put(Array.from(updated).map((id) => decode(contentRecords[id])));
```

Comment:
a lot of this looks like just moving code around, is this the main fix here?

Reply:
i did reorganize some things, but somehow the update method got removed.